detection-rules/body_microsoft_logo_bing_redirect.yml

```
@@ -42,9 +42,9 @@ source: |
            )
     )
   )
-
+  
   // Bing redirect
-  and any(body.links, .href_url.domain.root_domain == 'bing.com' and .href_url.path =~ '/ck/a')
+  and any(body.links, any(.href_url.rewrite.encoders, strings.contains(., "bing_open_redirect")))
```

we should handle both, for hunting's sake since older MDMs don't have this
```suggestion
  and any(body.links,
      (.href_url.domain.root_domain == 'bing.com' and .href_url.path =~ '/ck/a')
      or "bing_open_redirect" in .href_url.rewrite.encoders
  )
```